dist/css/main.css

```diff
@@ -0,0 +1,1176 @@
+  font-weight: 700;
+  opacity: 0.9;
+}
+
+.main-topic h4 {
+  font-size: 1rem;
+  font-weight: 400;
+  opacity: 0.6;
+}
+
+@media screen and (min-width: 768px) {
+  .main-topic.open {
+    width: 55vw;
+  }
+  .main-topic-topic {
+    padding: 3.5rem 1rem;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .main-content.open {
+    width: 55vw;
+  }
+}
+
+.main-content__box {
+  padding: 2rem 1rem;
+}
+
+.main-content__box .box-heading {
+  font-size: 1.75rem;
+  font-weight: 700;
+  opacity: 0.9;
+}
+
+.main-content__box .box-description {
+  font-size: 1rem;
+  opacity: 0.8;
+  margin-bottom: 2rem;
+}
+
+.main-content__box .box-description ul {
+  padding: 1rem 0 1rem 2rem;
+}
+
+.main-content__box .box-description ul li {
+  padding: 0.5rem 0;
+}
+
+.main-content__box .box-description code {
+  font-size: 0.875rem;
+  color: #d63384;
+  word-wrap: break-word;
+  padding-right: 0.25rem;
+}
+
+.display-code {
+  margin: 2rem 0;
+  padding-bottom: 2rem;
+  max-width: 375px;
+  border-bottom: 1px solid #d7d7d7;
+}
+
+@media screen and (min-width: 768px) {
+  .display-code {
+    margin: 2rem 0 4rem 0;
+    padding-bottom: 4rem;
+    max-width: 50vw;
+  }
+}
+
+.display-code iframe {
+  max-width: 100%;
+}
+
+/* Main Content Styles End */
+/* Side Navbar Styles Start */
+.menu-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 4rem;
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+  border-bottom: 1px solid #d7d7d7;
+  padding: 1rem;
+  z-index: 500;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu-logo a {
+  color: #ff3f6c;
+  font-size: 1.75rem;
+}
+
+.menu-btn {
+  position: relative;
+  height: 20px;
+  width: 28px;
+  cursor: pointer;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu-btn__burger {
+  position: absolute;
+  right: 0;
+  top: 0.5rem;
+  width: 28px;
+  height: 3px;
+  background: #272727;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu-btn__burger::before {
+  content: '';
+  position: absolute;
+  top: -8px;
+  width: 28px;
+  height: 3px;
+  background: #272727;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu-btn__burger::after {
+  content: '';
+  position: absolute;
+  top: 8px;
+  width: 20px;
+  height: 3px;
+  background: #272727;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu-btn__burger.open {
+  transform: rotate(720deg);
+  background: transparent;
+}
+
+.menu-btn__burger.open::before {
+  transform: rotate(45deg) translate(5px, 8px);
+  background: #272727;
+}
+
+.menu-btn__burger.open::after {
+  width: 28px;
+  transform: rotate(-45deg) translate(3px, -7px);
+  background: #272727;
+}
+
+.sidenav {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  right: 0;
+  background-color: #fff;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  padding-top: 5.75rem;
+  transition: all 0.5s ease-in-out;
+  border-left: 1px solid #d7d7d7;
+}
+
+.sidenav.open {
+  width: 65vw;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
Same here. I feel you can put everything in one media query for `min-width:768`. Curious to know your thinking here.

Reply:
Hey,
I think this is because of SCSS compiler and maybe on the way I've written out the code. My SCSS code for above files is:
```css
&-content {
    @include media-md {
      &.open {
        width: 55vw;
      }
    }
}
```
```css
&-topic {
    @include media-md {
      &.open {
        width: 55vw;
      }

      &-topic {
        padding: 3.5rem 1rem;
      }
    }
  } 
  ```
  
So, this is called `inline media queries` and the reason I wrote it out this way was that it made it much easier for me to style across screen sizes without leaving the component. 
Hope it makes sense! :D